package.json

```diff
@@ -319,7 +319,7 @@
     "karma-webpack": "^3.0.5",
     "mocha": "^5.2.0",
     "pre-commit": "^1.2.2",
-    "prettier": "1.18.2",
+    "prettier": "1.19.0",
```

Comment:
@Leowisd can we actually upgrade to the latest patch version, in this case, 1.19.1?

Reply:
Oh, sure.